1.6/windows/nanoserver/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM microsoft/nanoserver
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+# ideally, this would be C:\go to match Linux a bit closer, but C:\go is the recommended install path for Go itself on Windows
+ENV GOPATH C:\\gopath
+
+# PATH isn't actually set in the Docker image, so we have to set it from within the container
+RUN Write-Host 'Updating PATH ...'; \
+	$regPath = 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment'; \
+	\
+	$oldPath = ( \
+	Get-ItemProperty \
+		-Path $regPath \
+		-Name PATH \
+	).path; \
+	Write-Host ('  Before: {0}' -f $oldPath); \
+	\
+	$newPath = $env:GOPATH + '\bin;C:\go\bin;' + $oldPath; \
+	Write-Host ('  After:  {0}' -f $newPath); \
+	\
+	Set-ItemProperty \
+		-Path $regPath \
+		-Name PATH \
+		-Value $newPath; \
+	Write-Host 'Complete.';
```

Comment:
@tianon setx also works (it's a cmd.exe command, but should be in powershell too): https://github.com/dotnet/dotnet-docker/blob/master/1.0.0-preview2/nanoserver/Dockerfile#L13